src/Rook.py

```diff
@@ -27,6 +28,7 @@ def getPossibleMoves(self) -> Iterator[Move]:
 
         directions = [C(0, 1), C(0, -1), C(1, 0), C(-1, 0)]
         for direction in directions:
-            for move in self.movesInDirectionFromPos(currentPosition,
-                                                     direction, self.side):
+            for move in self.movesInDirectionFromPos(
+                currentPosition, direction, self.side
+            ):
```

Comment:
I guess this is debatable but I don't have a better option to offer, so it's ok.